Gemfile.lock

```diff
@@ -565,7 +565,7 @@ GEM
     runger_config (5.2.0)
       activesupport (>= 7.1.2)
     runger_email_reply_trimmer (0.3.0)
-    runger_style (2.16.0)
+    runger_style (2.17.0)
```

Comment:
This tells RuboCop to ignore some of the otherwise too-long lines in `annotate` comments about an index.